### PR TITLE
BZ 1555929 - Update Ceph RBD example

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1555929

### CODE REVIEW

File: install_config/storage_examples/ceph_example.adoc [hunk 1]
```diff
@@ -26,79 +26,93 @@ storage.
...
-value for the *client.admin* user:
+. Convert the keyring to base64:
++
+[source, bash]
+----
+$ /etc/ceph/ceph.client.openshift-test-keyring
+[client.openshift-test]
+	key = AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ==
+$ echo AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ== |base64
+QVFCTzJ4ZGJxM2lURGhBQVdDbzhsb1ZBNVJpdGxGNFNkZTV1WVE9PQo=
```
Comment (jhou1) [inline install_config/storage_examples/ceph_example.adoc]: echo -n AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ== |base64
QVFCTzJ4ZGJxM2lURGhBQVdDbzhsb1ZBNVJpdGxGNFNkZTV1WVE9PQ==

**Comment by jhou1** [comment]
In the **PV object definition using Ceph RDB example** section, the image `ceph-image` has to be created from the ceph side.

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/storage_examples/ceph_example.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [[install-config-storage-examples-ceph-example]]
-= Complete Example Using Ceph RBD
+= Using Ceph RBD for Persistent Storage
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: Why capitalize "Persistent Storage?"
Please update the title in the topics.yml file.

File: install_config/storage_examples/ceph_example.adoc [hunk 3]
```diff
@@ -26,79 +26,93 @@ storage.
 
 [NOTE]
 ====
-All `oc ...` commands are executed on the {product-title} master host.
+All `oc` commands are executed on the {product-title} master host.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: Should be active.

File: install_config/storage_examples/ceph_example.adoc [hunk 4]
```diff
@@ -26,79 +26,93 @@ storage.
...
-[[using-ceph-rbd-installing-the-ceph-common-package]]
-== Installing the ceph-common Package
-The *ceph-common* library must be installed on *all schedulable* {product-title} nodes:
-
-[NOTE]
-====
-The {product-title} all-in-one host is not often used to run pod workloads and,
+[discrete]
+== Prerequisites
+* Install the latest ceph-common package using `yum install -y ceph-common`. The
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: I still question if these are prereqs or step 1.

File: install_config/storage_examples/ceph_example.adoc [hunk 5]
```diff
@@ -26,79 +26,93 @@ storage.
...
+*ceph-common* library must be installed on *all schedulable* {product-title}
+nodes.
+* The {product-title} all-in-one host is not often used to run pod workloads and,
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: This isn't really a prereq.

File: install_config/storage_examples/ceph_example.adoc [hunk 6]
```diff
@@ -26,79 +26,93 @@ storage.
...
+$ /etc/ceph/ceph.client.openshift-test-keyring
+[client.openshift-test]
+	key = AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ==
+$ echo -n AQBO2xdbq3iTDhAAWCo8loVA5RitlF4Sde5uYQ== |base64
+QVFCTzJ4ZGJxM2lURGhBQVdDbzhsb1ZBNVJpdGxGNFNkZTV1WVE9PQ==
+----
 
-.Ceph Secret Definition
-====
+.  Create the ceph-secret.yaml with the base64-decoded keyring:
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: s/ceph-secret.yaml/*_ceph-secret.yaml_* file

File: install_config/storage_examples/ceph_example.adoc [hunk 7]
```diff
@@ -26,79 +26,93 @@ storage.
...
-  key: QVFBOFF2SlZheUJQRVJBQWgvS2cwT1laQUhPQno3akZwekxxdGc9PQ== <1>
-
+  key: QVFCTzJ4ZGJxM2lURGhBQVdDbzhsb1ZBNVJpdGxGNFNkZTV1WVE9PQo= <1>
 ----
-<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph auth get-key client.admin | base64` command, then copying the output and pasting it as the secret key's value.
-====
-
-Save the secret definition to a file, for example *_ceph-secret.yaml_*,
-then create the secret:
+<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: I'd update this note in the same way as the other topic.
And I still think that the yaml goes before the command to create the object. (all instances)

File: install_config/storage_examples/ceph_example.adoc [hunk 8]
```diff
@@ -125,85 +139,77 @@ single user (non-shared storage).
 <4> This defines the volume type being used. In this case, the *rbd* plug-in is
 defined.
 <5> This is an array of Ceph monitor IP addresses and ports.
-<6> This is the Ceph secret, defined above. It is used to create a secure
+<6> `ceph-image` must be created on the Ceph cluster.
+<7> This is the Ceph secret, defined above. It is used to create a secure
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: s/This is the Ceph secret, defined above./Enter the Ceph secret that you created.

File: install_config/storage_examples/ceph_example.adoc [hunk 9]
```diff
@@ -125,85 +139,77 @@ single user (non-shared storage).
...
+      storage: 2Gi <.2>
 ----
-<1> As mentioned above for PVs, the `*accessModes*` do not enforce access right,
-but rather act as labels to match a PV to a PVC.
-<2> This claim will look for PVs offering *2Gi* or greater capacity.
-====
-
-Save the PVC definition to a file, for example *_ceph-claim.yaml_*,
-and create the PVC:
+<1> As mentioned above for PVs, the `accessModes` do not enforce access right, but instead act as labels to match a PV to a PVC.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: Remove "As mentioned above for PVs,"
no comma after right

File: install_config/storage_examples/ceph_example.adoc [hunk 10]
```diff
@@ -125,85 +139,77 @@ single user (non-shared storage).
...
+<2> This claim looks for PVs offering 2Gi or greater capacity.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: s/offering/that offer

File: install_config/storage_examples/ceph_example.adoc [hunk 11]
```diff
@@ -313,5 +320,4 @@ objects:
     kubernetes.io/rbd
 ...
 ----
-<1> Place your super secret Ceph user key here in base64 format. See xref:using-ceph-rbd-creating-the-ceph-secret[Creating the Ceph Secret].
-====
+<1> Place your super secret Ceph user key here in base64 format.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: I'd remove "super secret."

File: install_config/storage_examples/ceph_example.adoc [hunk 12]
```diff
@@ -125,85 +139,77 @@ single user (non-shared storage).
 <4> This defines the volume type being used. In this case, the *rbd* plug-in is
 defined.
 <5> This is an array of Ceph monitor IP addresses and ports.
-<6> This is the Ceph secret, defined above. It is used to create a secure
+<6> `ceph-image` must be created on the Ceph cluster.
+<7> This is the Ceph secret, defined above. It is used to create a secure
 connection from {product-title} to the Ceph server.
-<7> This is the file system type mounted on the Ceph RBD block device.
-====
+<8> This is the file system type mounted on the Ceph RBD block device.
```
Comment (kalexand-rh) [inline install_config/storage_examples/ceph_example.adoc]: s/file system type mounted/type of file system that is mounted

**Pull request merged**